tests/FixedArrayTest.php

```diff
@@ -166,6 +166,61 @@ public function providerShiftDown() : array
         ];
     }
 
+    public function testCreate()
+    {
+        $fixedArray = FixedArray::create(4);
+
+        $this->assertSame(4, $fixedArray->getSize());
+        $this->assertInstanceOf(FixedArray::class, $fixedArray);
```

Comment:
Could you please move the `assertInstanceOf()` above `assertSame()`? Sounds more logical to test that it's actually an instance of, before calling any method on it.

Reply:
It doesn't mater that the order of assertions.
If you care about that it will be the logical tests, I will accept this change :).

Reply:
It does matter: when an assertion fails, it doesn't move to the next one. If we're going to call a method on a instance of a class, it's better to have checked *before* if it's such an instance.